src/main/java/io/tackle/controls/resources/StakeholderUpdateResource.java

```diff
@@ -0,0 +1,119 @@
+import io.tackle.controls.entities.StakeholderGroup;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.links.LinkResource;
+
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static javax.ws.rs.core.Response.Status.NO_CONTENT;
+
+@Path("stakeholder")
+public class StakeholderUpdateResource {
+
+    @Inject
+    Logger logger;
+/*
+    These are the synthetic methods create from Quarkus REST Panache.
+    They manage the 'PUT' verb both for creating and updating Stakeholder.
+    In this implementation I've removed the creation part to let it be done using the 'POST' verb.
+    
+    @Path("/{id}")
+    @Transactional
+    @PUT
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @LinkResource(
+            entityClassName = "io.tackle.controls.entities.Stakeholder",
+            rel = "update"
+    )
+    public Response update(@PathParam("id") Long var1, Stakeholder var2) {
+        StakeholderResourceImpl_cdf008bf94ad72ddbf067ca9cadc332df33146e6 var3 = this.resource;
+        if (var3.get(var1) == null) {
+            Object var4 = var3.update(var1, var2);
+            String var5 = (new ResourceLinksProvider()).getSelfLink(var4);
+            if (var5 != null) {
+                URI var7 = URI.create(var5);
+                Response.ResponseBuilder var6 = Response.status(201);
+                var6.entity(var4);
+                var6.location(var7);
+                return var6.build();
+            } else {
+                throw (Throwable)(new RuntimeException("Could not extract a new entity URL"));
+            }
+        } else {
+            var3.update(var1, var2);
+            return Response.status(204).build();
+        }
+    }
+
+    @Path("/{id}")
+    @Transactional
+    @PUT
+    @Consumes({"application/json"})
+    @Produces({"application/hal+json"})
+    public Response updateHal(@PathParam("id") Long var1, Stakeholder var2) {
+        StakeholderResourceImpl_cdf008bf94ad72ddbf067ca9cadc332df33146e6 var3 = this.resource;
+        if (var3.get(var1) == null) {
+            Object var4 = var3.update(var1, var2);
+            HalEntityWrapper var6 = new HalEntityWrapper(var4);
+            String var5 = (new ResourceLinksProvider()).getSelfLink(var4);
+            if (var5 != null) {
+                URI var8 = URI.create(var5);
+                Response.ResponseBuilder var7 = Response.status(201);
+                var7.entity(var6);
+                var7.location(var8);
+                return var7.build();
+            } else {
+                throw (Throwable)(new RuntimeException("Could not extract a new entity URL"));
+            }
+        } else {
+            var3.update(var1, var2);
+            return Response.status(204).build();
+        }
+    }
+*/
```

Comment:
Is there a reason to keep this commented code? Can we remove it?

Reply:
TBH I wanted to put it to have a reference to the auto-generated code the new method is replacing because, in the end, the autogenerated methods were not committed so we have no history of how it worked before.

Reply:
Thanks for clarifying it Marco. Let's keep the comments then